main.go

```diff
@@ -74,6 +74,13 @@ func main() {
 		}),
 	)
 
+	ap := startAPM(ctx, cfgs)
+	if ap == nil {
+		message := "Failed to start APM!"
+		fmt.Printf(message)
+		panic(message)
```

Comment:
startAPM already has a panic, we don't have to check the same here again. Can revert to the previous implementation